lib/GoClientGenerator.php

```diff
@@ -397,7 +397,7 @@ function writeContainer($currentClass, $className, &$isNeedJson)
 			$s .= "	}\n";
 			$s .= "	return nil\n";
 			$s .= "}\n";
-			$s .= "func (o *".$className."Container) MarshalJSON() ([]byte, error) {\n";
+			$s .= "func (o ".$className."Container) MarshalJSON() ([]byte, error) {\n";
```

Comment:
should we remove `*` in other places: UnmarshalJSON, Get?

Reply:
this is the only place that we found the issue, but im waiting for @gilurm answer maybe we dont need this change at all